README.md

```diff
@@ -47,26 +47,19 @@ npm start
 
 #### Trading
 
-* `CONNECTOR_LEDGERS` (default: `[]`) Ledgers where this connector has accounts. Used to auto-generate `CONNECTOR_PAIRS`.
-```js
-[
-  "USD@example.usd-ledger.",
-  "EUR@example.eur-ledger."
-]
-```
-* `CONNECTOR_CREDENTIALS` (default: `{}`) Connector's login credentials for various ledgers, ex.
+* `CONNECTOR_LEDGERS` (default: `{}`) Connector's login credentials for ledgers where it has accounts. Used to auto-generate `CONNECTOR_PAIRS`.
 ```js
 {
   // Using Basic Auth
-  "<ledger_address>": {
+  "USD@example.usd-ledger.": {
```

Comment:
We should get rid of the `USD@` part and just put a key in the object for `currency`